daemon/daemon_unix.go

```diff
@@ -1018,7 +1018,9 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *config.Co
 		ipamV6Conf     *libnetwork.IpamConf
 	)
 
-	if config.BridgeConfig.FixedCIDRv6 != "" {
+	if config.BridgeConfig.EnableIPv6 && config.BridgeConfig.FixedCIDRv6 == "" {
+		return fmt.Errorf("Please specify a IPv6 subnet using --fixed-cidr-v6 when IPv6 is enabled on the default bridge")
```

Comment:
Can you use `github.com/pkg/errors` to create the error and probably wrap this in `errdefs.InvalidParameter`?

Reply:
While at it, could you slightly update the error to [avoid "please"](https://github.com/docker/cli/issues/770)?

(giving this a go, but open to better suggestions);

```suggestion
		return fmt.Errorf("IPv6 is enabled for the default bridge, but no subnet is configured. Specify an IPv6 subnet using --fixed-cidr-v6")
```